webui/react/src/components/ColumnsCustomizeModal.test.tsx

```diff
@@ -222,15 +229,11 @@ describe('Columns Customize Modal', () => {
       DEFAULT_COLUMNS.length,
     );
 
-    const lineHeight = parseInt(within(lists[0]).getAllByRole('listitem')[0].style.height);
-
     await user.click(await view.findByText('Add All'));
 
     await waitFor(() => {
-      expect(parseInt(lists[0].style.height)).toEqual(0);
-      expect(parseInt(lists[1].style.height)).toEqual(
-        (NUM_GENERATED_COLUMNS + DEFAULT_COLUMNS.length) * lineHeight,
-      );
+      expect(lists[0].children[0].children[0].children[0].childElementCount).toEqual(0);
+      expect(lists[1].children[0].children[0].children[0].childElementCount).toBeGreaterThan(0);
```

Comment:
nit: these are some nasty selectors, can you look into some way of testing this that doesn't require drilling so deep into child elements?

Reply:
used `querySelector` to refactor this code